public/css/userGretting.module.css

```diff
@@ -0,0 +1,36 @@
+.greetingWrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+
+  & button {
+    font-size: 20px;
+    border: none;
+    transition: 100ms;
+    background-color: rgba(255, 255, 255, 0);
+    color: #fff;
+    font-family: poke-font;
```

Comment:
if all the buttons in the website will automatically change to default, put a `button { font-family: poke-font }` in the general css file

Reply:
you cant do tag selectors in the root of module.css. only classes and id's.